sweep/sweeper.go

```diff
@@ -645,6 +645,12 @@ func (s *UtxoSweeper) collector(blockEpochs <-chan *chainntnfs.BlockEpoch) {
 
 			// If this input is forced, we perform an sweep
 			// immediately.
+			//
+			// TODO(ziggie): Make sure when `immediate` is selected
+			// as a parameter that we only trigger the sweeping of
+			// this specific input rather than triggering the sweeps
+			// of all current pending inputs registered with the
+			// sweeper.
```

Comment:
👀 Is this an *eventual* TODO? or a TODO for *this PR*?

Reply:
This TODO is planned for a follow up PR, just marked it in the code base so we have it on the radar.